docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -36,109 +41,118 @@ $ sudo snap install edgexfoundry --channel=2.0
+```bash 
+sudo snap restart edgexfoundry.core-data
+```
 
-For details on the mapping of configuration options to Config options, please refer to [Service Environment Configuration Overrides](https://github.com/edgexfoundry/edgex-go/blob/main/snap/README.md#configuration-overrides). For details on configuration overrides for App Service Configurable or device service snaps, please refer to the snap documentation for each. Ex.
+**Note** - at this time changes to configuration values in the `[Writable]` section are not supported.
 
-`https://github.com/edgexfoundry/device-camera-go/blob/main/snap/README.md`
+For details on the mapping of configuration options to Config options, please refer to [Service Environment Configuration Overrides](https://github.com/edgexfoundry/edgex-go/blob/main/snap/README.md#configuration-overrides). 
+For details on configuration overrides please refer to documentations for [app service configurable snap](https://github.com/edgexfoundry/app-service-configurable/blob/main/snap/README.md#using-the-edgex-app-service-configurable-snap) or [device service snaps](#device-service-snaps).
 
 ## Viewing logs
 To view the logs for all services in an EdgeX snap use the `snap log` command:
 
 ```bash
-$ sudo snap logs edgexfoundry
+sudo snap logs edgexfoundry
 ```
 
 Individual service logs may be viewed by specifying the service name:
 
 ```bash
-$ sudo snap logs edgexfoundry.consul
+sudo snap logs edgexfoundry.consul
 ```
 
 Or by using the systemd unit name and `journalctl`:
 
 ```bash
-$ journalctl -u snap.edgexfoundry.consul
+journalctl -u snap.edgexfoundry.consul
 ```
 
-These techniques can be used with any snap including application and device services snaps.
+These techniques can be used with any snap including application snap and device services snaps.
 
 ## Security services
-The EdgeX snaps have security (Secret Store and API Gateway) enabled by default. As of Ireland, the security services in the
-edgexfoundry snap consitute the following components:
 
-    - kong
-    - postgres
-    - vault
-    - security-bootstrapper-redis**
-    - security-consul-bootstrapper**
-    - security-secretstore-setup**
-    - security-proxy-setup**
+Currently, The EdgeX snap has security (Secret Store and API Gateway) enabled by default. The security services constitute the following components:
+
+    - kong-daemon (API Gateway a.k.a. Reverse Proxy)
+    - postgres (kong's database)
+    - vault (Secret Store)
+
+Oneshot services which perform the necessary security setup and stop, when listed using `snap services`, they show up as `enabled/inactive`:
 
-** these services are all 'oneshot' services, when listed using `snap services`, they show up as `enabled/inactive`.
+    - security-proxy-setup (kong setup)
+    - security-secretstore-setup (vault setup)
+    - security-bootstrapper-redis (secure redis setup)
+    - security-consul-bootstrapper (secure consul setup)
```

Comment:
Remove indentation. These are recognized as code blocks in the rendered version.

Reply:
Removed.